packages/core/src/helpers.ts

```diff
@@ -322,4 +318,24 @@ export default class Helpers {
         options.uriVariables = uriVariables;
         return options;
     }
+
+    public static validateInteractionOptions(
```

Comment:
I would return a boolean. Usually, I use exceptions only for unexpected results, but here we are actually expecting that the interaction option might not be valid.

Reply:
The method would only return a true boolean then (?) otherwise it would throw instead of returning false. Is this behavior ok?

Reply:
IMHO, it should return `false` instead of throwing then in parseInteractionOptions if it returns false you throw.